app/graphql/resolvers/corpus_slate_lineup_resolvers.py

```diff
@@ -66,7 +66,7 @@ async def resolve_home_slate_lineup(root, info: Info, locale: str = 'en-US') ->
 
         slate_lineup_model = await HomeDispatch(
             corpus_client=di.corpus_feature_group_client,
-            preferences_provider=UserRecommendationPreferencesProvider(di.aioboto3_session, topic_provider),
+            preferences_provider=UserRecommendationPreferencesProviderV2(di.aioboto3_session, topic_provider),
```

Comment:
Should we rename `UserRecommendationPreferencesProviderV2` to `UserRecommendationPreferencesProvider` as future devs may not care, or know about the version history?